### PR TITLE
spin and flip are fun again

### PR DESCRIPTION
**Pull request by KangTut**
## About The Pull Request
Removes the dumb cool downs from the spin and flip emotes. You won't fall down or get dizzy anymore.
## Why It's Good For The Game
Because spinning and flipping do nothing but look funny

### CODE REVIEW

**Comment by karmaisblackandbluepilled** [comment]
idk they make combat way more aids because you can set them to happen on every button press

File: code/modules/mob/emote.dm [hunk 1]
```diff
@@ -99,29 +77,3 @@
...
-		if(current_confusion < BEYBLADE_CONFUSION_LIMIT)
-			user.add_confusion(BEYBLADE_CONFUSION_INCREMENT)
-
-
-#undef BEYBLADE_PUKE_THRESHOLD
-#undef BEYBLADE_PUKE_NUTRIENT_LOSS
-#undef BEYBLADE_DIZZINESS_PROBABILITY
-#undef BEYBLADE_DIZZINESS_VALUE
-#undef BEYBLADE_CONFUSION_INCREMENT
-#undef BEYBLADE_CONFUSION_LIMIT
```
Comment (TheChosenEvilOne) [inline code/modules/mob/emote.dm:127]: remove the #defines as well

**Comment by KangTut** [comment]
I don't recall ever seeing anyone use it in combat to get an advantage. I guess it could make you a little harder to click directly but I don't see an issue with that.

**Review by TheChosenEvilOne — APPROVED** [review]

**Comment by karmaisblackandbluepilled** [comment]
The moment someone ported these to hippie, a guy put all his buttons to spin and flip. I will also do this from now on.

**Pull request merged**